programs-remote-listings/rs-connect.php

```diff
@@ -120,7 +120,7 @@ public function register_query_var($vars)
     public function insert_shortcode($content)
     {
         if ($this->configured()) {
-            $current_page = $GLOBALS['post']->post_name;
+            $current_page = $GLOBALS['post']->post_name ?? null;
```

Comment:
Does`$GLOBALS['post']` always exist?   No need for: `$GLOBALS['post']?->post_name ?? null;`. (or whatever the php 7 equivalent is)